index.html

```diff
@@ -30,5 +30,82 @@ <h2>I’am a Software Developer</h2>
         <div class="illustration"><img src="images/illustration-header-mobile.svg" alt="illustration"></div>
```

Comment:
Kindly. Layout images should be placed in the background of your site, I recommend using background-image.

Reply:
Kindly. links in the A tags, if they don't exist they should put a # to avoid any errors.